spec/models/line_item_spec.rb

```diff
@@ -22,5 +22,13 @@
 		it "requires a quantity" do
 			expect(build(:line_item, quantity: nil)).not_to be_valid
 		end
+
+		it "the quantity must be an integer and cannot be 0" do
+			expect(build(:line_item, quantity: 'a')).not_to be_valid
+			expect(build(:line_item, quantity: '1.0')).not_to be_valid
```

Comment:
I wonder if this would still be not valid without the `only_integer`
...
...
I'll stop thinking about this now.